recipes/rrapply/meta.yaml

```
@@ -0,0 +1,80 @@
+{% set version = '1.2.1' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-rrapply
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/rrapply_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/rrapply/rrapply_{{ version }}.tar.gz
+  sha256: d1c63e2b0913e014aaf162f5d27ec305cde35c8237b4e3636f7aea089749ab38
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}make
+    - {{ posix }}sed               # [win]
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+  host:
+    - r-base
+  run:
+    - r-base
+    - {{ native }}gcc-libs         # [win]
+
+test:
+  commands:
+    - $R -e "library('rrapply')"           # [not win]
+    - "\"%R%\" -e \"library('rrapply')\""  # [win]
+
+about:
+  home: https://jorischau.github.io/rrapply/, https://github.com/JorisChau/rrapply
+  license: GPL-2.0-or-later
```

```suggestion
  license: GPL-3.0-only
```

cran meta data tells `# License: GPL-3` which is the spdx short `GPL-3.0-only`